packages/@okta/vuepress-theme-prose/components/TermsAndConditionsDialog.vue

```diff
@@ -9,7 +9,7 @@
         experience. Your personal data is safe and secure with us.
       </p>
       <label for="tac-country">
-        Country
+        Country/Region: 
```

Comment:
does it require of using colon here?

Reply:
good catch, we do not need the colon if it wasn't there before